doc/sphinxman/source/build_planning.rst

```diff
@@ -0,0 +1,1152 @@
+Run |PSIfour|. ::
+
+    >>> cat sample.py
+    import psi4
+    mol = psi4.geometry("""
+    He
+    """)
+    psi4.energy('hf/cc-pvdz')
+    psi4.compare_values(-2.85518839, psi4.core.get_variable('current energy'), 5, 'SCF E')
+    >>> python sample.py
+    SCF E.............................................................PASSED
+
+
+.. _`faq:runfrombinary`:
+
+How to run Psi4 as executable or Python module from conda installation
+----------------------------------------------------------------------
+
+The configuration commands below are generic versions of the ones printed
+to your screen as advice upon installing |PSIfour| into a Anaconda,
+Miniconda, or Psi4conda distribution, :samp:`{condadist} =
+{ana|mini|psi4}conda`. If you hadn't already done so, issue the following
+commands directly in your terminal or place them into your "rc" file and
+open a new terminal.
+
+If you installed the Psi4conda distribution or installed the |PSIfour|
+conda package into the main environment of an Anaconda or Miniconda
+distribution and added that to your :envvar:`PATH`, as prompted, then
+``which psi4`` likely yields :samp:`{condadist}/bin/psi4` and the ``PATH``
+setting lines below are redundant.
+
+If you installed into a conda environment :samp:`{p4env}` and performed
+:samp:`source activate {p4env}`, then ``which psi4`` likely yields
+:samp:`{condadist}/envs/{p4env}/bin/psi4` and the ``PATH`` setting lines
+below are redundant.
+
+.. code-block:: tcsh
+
+    # csh, tcsh: add to shell or ~/.tcshrc file
+    unsetenv PSIDATADIR
+    setenv PATH {prefix}/bin:$PATH
+    setenv PSI_SCRATCH /path/to/existing/writable/local-not-network/directory/for/scratch/files
+
+.. code-block:: bash
+
+    # sh, bash: add to shell or ~/.bashrc (Linux) or ~/.bash_profile (Mac) file
+    unset PSIDATADIR
+    export PATH={prefix}/bin:$PATH
+    export PSI_SCRATCH=/path/to/existing/writable/local-not-network/directory/for/scratch/files
+
+.. If you installed the Psi4conda distribution or installed the |PSIfour|
+.. conda package into the main environment of an Anaconda or Miniconda
+.. distribution and added that to your :envvar:`PATH`, as prompted, then
+.. :samp:`{condadist} = $HOME/{ana|mini|psi4}conda` and ``which psi4`` likely
+.. yields :samp:`{condadist}/bin/psi4` and the ``PATH`` setting lines above
+.. are redundant.
+
+Run |PSIfour| as executable. ::
+
+    >>> cat sample.in
+    molecule {
+    He
+    }
+    energy('hf/cc-pvdz')
+    compare_values(-2.85518839, get_variable('current energy'), 5, 'SCF E')
+    >>> psi4 sample.in
+    SCF E.............................................................PASSED
+
+Or run |PSIfour| as Python module. ::
+
+    >>> cat sample.py
+    import psi4
+    mol = psi4.geometry("""
+    He
+    """)
+    psi4.energy('hf/cc-pvdz')
+    psi4.compare_values(-2.85518839, psi4.core.get_variable('current energy'), 5, 'SCF E')
+    >>> python sample.py
+    SCF E.............................................................PASSED
+
```

Comment:
Could this be too much duplication? Might be more straightforward to just point out that `conda` is handeling the paths for you.